v2/src/log_sink_console.c

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdio.h>
+
+#include "macro_utils/macro_utils.h"
+
+#include "c_logging/logger.h"
+#include "c_logging/log_context.h"
+
+#include "c_logging/log_sink_console.h"
+
+void log_sink_console_log(LOG_LEVEL log_level, LOG_CONTEXT_HANDLE log_context, const char* message, const char* file, const char* func, int line)
```

Comment:
>log_context [](http://example.com/codeflow?start=66&length=11)

seems to be allowed to be NULL (and that's good), but line 17 will dereference the NULL pointer.

Reply:
It's guarded by (log_context != NULL)